src/index.js

```diff
@@ -26,9 +26,9 @@ client.on('message', message => {
   }
 
   // Check that we are not in private messages
-  if (!channel) return;
+  if (channel.type !== 'text') return;
```

Comment:
you sure this (accessing the type property) will work if channel was null before? To be clear, I'm not sure if it was null

Reply:
just tested, and it looks like a DmChannel was returned with type dm. This is a good fix. Merging :)

Reply:
Yes, per the docs `Message.channel` will be any of `TextChannel`, `DMChannel`, or `GroupDMChannel`. The property should never be undefined.
Furthermore the bot would never receive a `Message` over a `GroupDMChannel` since group DMs require at least 1 user to be friends with the bot, and bots cannot have friends (there is no API endpoint for adding friends - it must be done in the Discord client by the user). Poor lonely bots!